### PR TITLE
[presentation-api] Update two tests for PresentationRequest

### PR DESCRIPTION
**Pull request by tomoyukilabs**
This PR updates a couple of tests for PresentationRequest:

- PresentationRequest_success.https.html:
  - `http://` in each presentation URL(s) is replaced with `https://`.
- PresentationRequest_onconnectionavailable-manual.https.html:
  - Terminating a connection in `t.add_cleanup` is improved.
  - `EventWatcher` is used to watch a `connectionavailable` event.
  - PresentationConnectionAvailableEvent is inspected as well.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5713)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226279972)

# Lint #

Passed

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226279974)

# Firefox (nightly channel) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226279980)

# Chrome (unstable channel) #

File: presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.https.html [hunk 1]
```diff
@@ -17,43 +17,57 @@
...
+            //       after the promise P is resolved (step 8).
+            return request.start().then(c => {
                 connection = c;
                 assert_equals(connection.state, 'connecting', 'The initial state of the presentation connection is "connecting".');
                 assert_true(!!connection.id, 'The connection ID is set.');
                 assert_true(typeof connection.id === 'string', 'The connection ID is a string.');
                 assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
+
+                const eventWatcher = new EventWatcher(t, request, 'connectionavailable');
+                const timeout = new Promise(resolve => { t.step_timeout(resolve, 5000); });
```
Comment (tidoust) [inline presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.https.html]: Minor suggestion to improve readability (well, for some definition of readability perhaps): for me, a timeout is something that makes the test fail, so I would translate that into a Promise that gets rejected. In other words, I would write something like:

```js
const timeout = new Promise((_, reject) => { t.step_timeout(() => {
  reject('The connectionavailable event was not fired (timeout)'
}, 5000); });
return Promise.race([ eventWatcher.wait_for('connectionavailable'), timeout ]); 
```

... and then no need to have the `if (!evt)` check in the `then` part.

**Review by tidoust — COMMENTED** [review]
Thanks! See very minor suggestion inline. Feel free to ignore!

File: presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.https.html (hunk 1 above)
Comment (tomoyukilabs) [inline presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.https.html]: In fact, I wondered which seemed better. Now it looks preferable, I'll modify the code later. Thanks!

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5713/)

**Review by tidoust — APPROVED** [review]
Great, thanks!

**Pull request merged**